client/src/components/VtkViewer.vue

```diff
@@ -364,8 +364,9 @@ export default {
       class="toolbar"
       dark
       flat
-      color="black"
-      max-height="42"
+      color="#ffffff00"
```

Comment:
I may be over-interpreting here; I'm assuming our intention is that the toolbar should be the same color as the container it's inside of. If that's not the case, we can back out the color change here.

Reply:
That's a good point and I like this better.